oeqLinkMigrationLTI/src/main/java/org/apereo/openequella/integration/blackboard/linkmigrationlti/FixerUtils.java

```diff
@@ -421,15 +426,60 @@ private void setNameAndDescription(FixerResponse resp, String textToSearch, int
 	}
   }
 
-  private String removeOldIntegLink(String str, int level, String method) {
-    final String start = "<td><table";
-    final String end = "</table></td></tr>";
-     final String toRemove = StringUtils.substringBetween(str, start, end);
-     if(toRemove.contains("href=\"/webapps/dych-tle-")) {
-       return StringUtils.replace(str,start+toRemove+end, "<td/></tr>");
-	 } else {
-       log(level+1, "Not removing old integration link from the body - could not find the 'dych-tle' table");
-       return str;
-	 }
+  private String scrubDescription(String str, int level, String[] urls) {
+    Document d = Jsoup.parse(str);
+    d.outputSettings().prettyPrint(false);
+
+    // Remove all old integration links
+    Elements elems = d.select("a[href]");
+    for(Element e : elems) {
+      if (e.attr("href").startsWith("/webapps/dych-tle-")) {
```

Comment:
I wonder if using a prefix selector (https://developer.mozilla.org/en-US/docs/Web/CSS/Attribute_selectors)

```
a[href^="/webapps/dych-tle-"]
```

could help simplify this step

Reply:
huh.  I thought I tried that.  Apparently not though.  :upside_down_face:   Switched to using the prefix selector.